t/Makefile.am

```diff
@@ -224,6 +224,7 @@ dist_check_SCRIPTS = \
 	issues/t2686-shell-input-race.sh \
 	issues/t3186-python-future-get-sigint.sh \
 	issues/t3415-job-shell-segfault-on-null.sh \
+	issues/t3432-python-sigint.sh \
```

Comment:
Minor nit on the commit message: `Add a test to t4000-issues-test-driver.t` -> `Add a test`

Reply:
These tests are all driven from `t4000-issues-test-driver.t`, which does
```sh
for testscript in ${FLUX_SOURCE_DIR}/t/issues/*; do
    testname=`basename $testscript`
    prereqs=$(sed -n 's/^.*test-prereqs: \(.*\).*$/\1/gp' $testscript)
    test_expect_success  "$prereqs" $testname "run_timeout 30 $testscript"
done

```

Reply:
I reworded the commit message to be more clear:

```
commit 6c311dd82da71a7b0c193ba8deceec563e76644e
Author: Mark A. Grondona <mark.grondona@gmail.com>
Date:   Fri Dec 18 12:55:42 2020 -0800

    testsuite: add reproducer for issue#3432
    
    Add a test to t/issues (which will be automatically run via
    t4000-issues-test-driver.t) that ensures that a Python program sleeping
    in the Flux reactor is interruptible, even if a reactor callback makes
    a synchronous RPC.

```

Reply:
Ah! Thanks for that clarification and sorry for the noise.